src/executables/csql_launcher.c

```diff
@@ -25,6 +25,7 @@
 
 #include <stdio.h>
 #include <stdarg.h>
+#include "config.h"
```

Comment:
Please move it to the top with the following blank line:
```c
#include "config.h"

#include <stdio.h>
...
```

This is our convention.